src/main/java/com/uid2/admin/vertx/service/SiteService.java

```diff
@@ -296,6 +301,40 @@ private void handleSiteDomains(RoutingContext rc) {
         }
     }
 
+    private void handleSiteCreated(RoutingContext rc) {
```

Comment:
I started out thinking this was handling a new site created action - changing the name to something like "handleUpdateSiteCreated" ior "handleUpdateCreatedTimestamp" would be clearer.